src/main/java/jsr223/docker/compose/utils/CommandlineOptionsFromBindingsExtractor.java

```diff
@@ -98,4 +105,40 @@
         options.put(OptionType.UP_OPTION, upCmdOptions);
         return options;
     }
+
```

Comment:
Looks good, I think you can remove the "old" code

Reply:
This file is shared by the two script engines. I keep it in the utils folder